modules/Project/Save.php

```diff
@@ -133,4 +133,4 @@
 	$_REQUEST['return_url'] = "index.php?module=Project&action=view_GanttChart&project_id=" . $return_id;
     handleRedirect($return_id,'Project');
 }
-?>
\ No newline at end of file
+?>
```

Comment:
Thanks for you guide... I tried to remove that line and it is removing when I commit but after that it shows the same line ...I dont know why its happening like that. Meanwhile I have commited with my another account. Please merge that https://github.com/nadiln/SuiteCRM/pull/1/commits/04e99d12fc8a576d8c9d1fbde8c1ab2cd936c486